connector/connect/server/src/main/scala/org/apache/spark/sql/connect/config/Connect.scala

```diff
@@ -121,4 +121,18 @@ object Connect {
       .version("3.5.0")
       .booleanConf
       .createWithDefault(false)
+
+  val CONNECT_USER_SESSION_CACHE_SIZE =
+    ConfigBuilder("spark.connect.userSession.cacheSize")
```

Comment:
The Spark Sessions are shared common or for user?

Reply:
It is common

Reply:
So, how about `spark.connect.session.cacheSize` ?

Reply:
It is ok, thanks